build.gradle

```diff
@@ -183,9 +183,9 @@ tasks.register("verifyVersions") {
  * after the backport of the backcompat code is complete.
  */
 
-boolean bwc_tests_enabled = false
```

Comment:
Should we just kill this part. Seems like a relic of breaking backwards compatibility tests, no?

Reply:
I think we might need it to disable the bwc checks temporarily from time to time as explained in the comment above.

Reply:
Well, it doesn't explain why. Why?

Reply:
Hmm, when it says "necessary", I think, if you don't disable it then until you backport the change to other major release branches, the checks will start failing in CI and that will stop you merging the code to main?

Maybe, let's just keep it for now and if we think this is not necessary we will remove it later.